integration-tests/sandbox-drain-delegate-operation.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { TezosToolkit } from "@taquito/taquito";
+import { CONFIGS, sleep } from "./config";
+
+CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
+  const Tezos = lib;
+  const flextesanet = (rpc === 'http://localhost:20001') ? it : it.skip;
```

Comment:
we could maybe specify which protocol it is by renaming this to `flextesaLimanet`

Reply:
thanks, updated to flextesaLimaNet